previsao-hieraquica.tex

```diff
@@ -395,7 +395,7 @@
 \addcontentsline{toc}{chapter}{Introdução}
 % ----------------------------------------------------------
 
-A etapa de previsão e projeção de demanda sempre figuraram como uma etapa importante em qualquer atividade de planejamento.
+A etapa de previsão de demanda sempre figuraram como uma etapa importante em qualquer atividade de planejamento.
```

Comment:
Because it is redundant with previsão. You should use them interchangeably to avoid repetition of words.